.github/workflows/chart-lint.yml

```diff
@@ -0,0 +1,70 @@
+name: Lint Helm Chart
+
+concurrency:
+  group: ci-${{ github.run_id }}
+  cancel-in-progress: true
```

Comment:
we should probably add concurrency to the `push-ci.yml` file too?

Reply:
make sense! Added now!